frontend/src/lib/components/CommandBar/CommandBar.tsx

```diff
@@ -21,13 +21,10 @@ const CommandBarOverlay = forwardRef<HTMLDivElement, CommandBarOverlayProps>(fun
 ): JSX.Element {
     return (
         <div
-            className="fixed top-0 left-0 w-full h-full flex flex-col items-center justify-center p-3"
-            // eslint-disable-next-line react/forbid-dom-props
-            style={{
-                zIndex: 'var(--z-command-palette)',
-                backgroundColor: 'var(--modal-backdrop-color)',
-                backdropFilter: 'blur(var(--modal-backdrop-blur))',
-            }}
+            className={`fixed top-0 left-0 w-full h-full flex flex-col items-center justify-center p-3
+                z-[var(--z-command-palette)]
+                background-[var(--modal-backdrop-color)]
+                backdrop-blur-[var(--modal-backdrop-blur)]`}
```

Comment:
We should not use dynamic class names as per the Tailwind docs: https://tailwindcss.com/docs/content-configuration#dynamic-class-names.

It's probably fine to leave this explicit skip in place or to create a `CommandBar__overlay` class in the associated index.scss file

Reply:
I'm not sure if this would be considered as dynamic class names (they are defined here statically in whole), but creating the new class looks like a good idea, thanks!

Reply:
Yeah, you're actually correct here. My bad